web/packages/teleport/src/Audit/fixtures/index.ts

```diff
@@ -2042,6 +2042,26 @@ export const events = [
     uid: '503e310d-8d88-4bea-bbbb-a1b35456a03a',
     user: 'alice',
   },
+  {
+    cluster_name: 'im-a-cluster-name',
+    code: 'TDB07I',
+    db_name: 'master',
+    db_protocol: 'postgres',
+    db_service: 'postgres-local',
+    db_uri: 'localhost:1433',
+    db_user: 'alice',
+    ei: 50,
+    event: 'db.session.malformed_packet',
```

Comment:
Why the event id is `db.session.malformed_packet` instead of `db.session.permissions.update` 

Could you attach in PR description  UI changes by capturing screenshot from the result produced by  `cd web` ` yarn storybook` -> `localhost:9002` -> `[Audit] Tab` -> `[All Possible Events]`

Reply:
Copy-pasta on my side. I'll fix and also share the screenshot.

Reply:
Fixed the typo. 

I've added a screenshot in a comment: https://github.com/gravitational/teleport/pull/37798#issuecomment-1934024309. I believe PR description goes as a commit message, so I didn't add it there.